arrows/serialize/json/tests/test_serialize.cxx

```diff
@@ -266,18 +266,35 @@ TEST( serialize, image)
   {
     *cp++ = i;
   }
+  {
```

Comment:
Consider adding duplicates of this test for some more complex image memory layouts.  Those are more likely to fail.  Create a view that is a crop of the original image and test that.  At the moment you need to manually use one of the more complex constructors to do this.

```
// 'offset' is some positive amount
// some of {w, h, d} should be smaller than the original size 
image( other.memory(), other.first_pixel() + offset,
             w, h, d, other.w_step(), other.h_step(), other.d_step(), other.pixel_format());
```

(As a side note, we should add helper functions to make it easier to do these crops.  I've only not added them because I don't have a good place to put them.  After restructuring KWIVER a bit I think we should have an image manipulation library in arrows)

Reply:
done